sim/core/spell_damage.go

```diff
@@ -171,18 +171,18 @@ func BaseDamageFuncMeleeWeapon(hand Hand, normalized bool, flatBonus float64, ba
 		} else {
 			return func(sim *Simulation, hitEffect *SpellEffect, spell *Spell) float64 {
 				damage := spell.Unit.AutoAttacks.OH.CalculateNormalizedWeaponDamage(sim, hitEffect.MeleeAttackPower(spell.Unit))
-				damage = damage*0.5*baseMultiplier + flatBonus
+				damage = damage + flatBonus
 				if includeBonusWeaponDamage {
 					damage += hitEffect.BonusWeaponDamage(spell.Unit)
 				}
-				return damage * weaponMultiplier
+				return damage * weaponMultiplier * 0.5
```

Comment:
For Mutilate, and the Sigil of Awareness case outlined in `death_strike.go`, I'd prefer to stay "as close to how the game (likely) works" as possible.

Reply:
I think you're confusing +Weapon Damage and Flat Bonus Damage (they are 2 different things). I will fix the 0.5 multiplier to not be applied to the +Weapon Damage but it will still be applied to the flat damage without having to create a special case for ToT

Reply:
Also Bonus Weapon Damage is still used by some tbc items/spells so we can't remove it right now

Reply:
> This is inconvenient for ToT, but only because it's modelled differently from how the game works (single spell vs. two different spells for MH and OH).

Just because they are different spells does not mean we should not re-use our own code. In the end looking at their values the OH spell is the same spell with a 50% flat modifier of the MH Spell. We are 100% sure NoCS affects it as this was tested and confirmed on beta so we should not handle it as a special case